### PR TITLE
Add third party library

### PR DESCRIPTION
**Pull request by mayank1513**
## Related Issues or Discussions

Fixes #

## Summary
Added a third party library under integrations

[zustand-sync-tabs](https://www.npmjs.com/package/zustand-sync-tabs) — Zustand middleware to easily sync Zustand state between tabs / windows / iframes (SameOrigin)

## Check List

- [x] `yarn run prettier` for formatting code and docs

### CODE REVIEW

File: docs/integrations/third-party-libraries.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ This can be done using third-party libraries created by the community.
 - [zustand-rx](https://github.com/patdx/zustand-rx) — A Zustand middleware enabling you to subscribe to a store as an RxJS Observable.
 - [zustand-saga](https://github.com/Nowsta/zustand-saga) — A Zustand middleware for redux-saga (minus redux).
 - [zustand-store-addons](https://github.com/Diablow/zustand-store-addons) — React state management addons for Zustand.
+- [zustand-sync-tabs](https://www.npmjs.com/package/zustand-sync-tabs) — Zustand middleware to easily sync Zustand state between tabs / windows / iframes (SameOrigin)
```
Comment (sewera) [inline docs/integrations/third-party-libraries.md:49]: Do you have a Github link? It was hard to found quickly from the NPM page, at least for me.

Most if not all links here are repository links, so I think at this point this is what most people expect.

**Review by sewera — CHANGES_REQUESTED** [review]

**Comment by dai-shi** [comment]
Please put GitHub link.

File: docs/integrations/third-party-libraries.md [hunk 2]
```diff
@@ -46,6 +46,7 @@ This can be done using third-party libraries created by the community.
 - [zustand-rx](https://github.com/patdx/zustand-rx) — A Zustand middleware enabling you to subscribe to a store as an RxJS Observable.
 - [zustand-saga](https://github.com/Nowsta/zustand-saga) — A Zustand middleware for redux-saga (minus redux).
 - [zustand-store-addons](https://github.com/Diablow/zustand-store-addons) — React state management addons for Zustand.
+- [zustand-sync-tabs](https://www.npmjs.com/package/zustand-sync-tabs) — Zustand middleware to easily sync Zustand state between tabs / windows / iframes (SameOrigin) [GitHub](https://github.com/mayank1513/zustand-sync-tabs)
```
Comment (dai-shi) [inline docs/integrations/third-party-libraries.md:49]: Can we do this?
```suggestion
- [zustand-sync-tabs](https://github.com/mayank1513/zustand-sync-tabs) — Zustand middleware to easily sync Zustand state between tabs / windows / iframes (SameOrigin)
```

File: docs/integrations/third-party-libraries.md [hunk 3]
```diff
@@ -46,6 +46,7 @@ This can be done using third-party libraries created by the community.
 - [zustand-rx](https://github.com/patdx/zustand-rx) — A Zustand middleware enabling you to subscribe to a store as an RxJS Observable.
 - [zustand-saga](https://github.com/Nowsta/zustand-saga) — A Zustand middleware for redux-saga (minus redux).
 - [zustand-store-addons](https://github.com/Diablow/zustand-store-addons) — React state management addons for Zustand.
+- [zustand-sync-tabs](https://github.com/mayank1513/zustand-sync-tabs) — Zustand middleware to easily sync Zustand state between tabs / windows / iframes (SameOrigin)
```
Comment (sewera) [inline docs/integrations/third-party-libraries.md:49]: You can add a full stop at the end and rephrase "SameOrigin" to something like "for apps from the same origin."

```suggestion
- [zustand-sync-tabs](https://github.com/mayank1513/zustand-sync-tabs) — Zustand middleware to easily sync Zustand state between tabs / windows / iframes (SameOrigin).
```

**Review by sewera — CHANGES_REQUESTED** [review]

File: docs/integrations/third-party-libraries.md [hunk 4]
```diff
@@ -46,6 +46,7 @@ This can be done using third-party libraries created by the community.
 - [zustand-rx](https://github.com/patdx/zustand-rx) — A Zustand middleware enabling you to subscribe to a store as an RxJS Observable.
 - [zustand-saga](https://github.com/Nowsta/zustand-saga) — A Zustand middleware for redux-saga (minus redux).
 - [zustand-store-addons](https://github.com/Diablow/zustand-store-addons) — React state management addons for Zustand.
+- [zustand-sync-tabs](https://github.com/mayank1513/zustand-sync-tabs) — Zustand middleware to easily sync Zustand state between tabs / windows / iframes (SameOrigin).
```
Comment (sewera) [inline docs/integrations/third-party-libraries.md:49]: Just like in a previous PR:

```suggestion
- [zustand-sync-tabs](https://github.com/mayank1513/zustand-sync-tabs) — Zustand middleware to easily sync Zustand state between tabs/windows/iframes with same origin.
```

**Review by sewera — CHANGES_REQUESTED** [review]

**Pull request merged**